BE/LegalEntities/FormalBusinessOrganizations.rdf

```diff
@@ -209,12 +213,63 @@
+		<rdfs:subClassOf>
+			<owl:Class>
+				<owl:unionOf rdf:parseType="Collection">
+					<rdf:Description rdf:about="&fibo-be-le-fbo;OrganizationIdentifier">
+					</rdf:Description>
+					<rdf:Description rdf:about="&fibo-fnd-aap-ppl;NationalIdentificationNumber">
+					</rdf:Description>
+				</owl:unionOf>
+			</owl:Class>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&lcc-lr;isMemberOf"/>
+				<owl:onClass rdf:resource="&fibo-be-le-fbo;TaxIdentificationScheme"/>
+				<owl:qualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">1</owl:qualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&lcc-lr;identifies"/>
+				<owl:onClass rdf:resource="&fibo-fnd-pty-pty;IndependentParty"/>
+				<owl:qualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">1</owl:qualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label>tax identifier</rdfs:label>
+		<skos:definition>identifier assigned to a taxpayer that enables compulsory financial charges and other levies to be imposed on the taxpayer by a governmental organization in order to fund government spending and various public expenditures</skos:definition>
+		<fibo-fnd-utl-av:adaptedFrom rdf:datatype="&xsd;anyURI">https://www.oecd-ilibrary.org/taxation/standard-for-automatic-exchange-of-financial-account-information-in-tax-matters-second-edition_9789264267992-en</fibo-fnd-utl-av:adaptedFrom>
+		<fibo-fnd-utl-av:explanatoryNote>Tax identifiers are used for various tax-related purposes in the United States and in other countries under the Common Reporting Standard (CRS).</fibo-fnd-utl-av:explanatoryNote>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-be-le-fbo;ValueAddedTaxIdentificationNumber">
+		<rdfs:subClassOf rdf:resource="&fibo-be-le-fbo;OrganizationIdentifier"/>
+		<rdfs:subClassOf rdf:resource="&fibo-be-le-fbo;TaxIdentifier"/>
+		<rdfs:label>value-added tax identification number</rdfs:label>
+		<skos:definition>tax identifier used in many countries that is assessed incrementally, levied on the price of a product or service at each stage of production, distribution, and sale to the end consumer</skos:definition>
+		<fibo-fnd-utl-av:abbreviation>VATIN</fibo-fnd-utl-av:abbreviation>
+		<fibo-fnd-utl-av:adaptedFrom rdf:datatype="&xsd;anyURI">https://ec.europa.eu/taxation_customs/business/vat/eu-vat-rules-topic/vat-identification-numbers_en</fibo-fnd-utl-av:adaptedFrom>
+		<fibo-fnd-utl-av:explanatoryNote>If the ultimate consumer is a business that collects and pays to the government VAT on its products or services, it can reclaim the tax paid.  Not all localities require VAT to be charged, and exports are often exempt. VAT is usually implemented as a destination-based tax, where the tax rate is based on the location of the consumer and applied to the sales price.</fibo-fnd-utl-av:explanatoryNote>
```

Comment:
The last sentence applies just as much to sales taxes that are not VAT

Reply:
Ok, but it is still true of VAT.  A sales tax is similar to a VAT in any case.